build-scripts/packageAlpha.ts

```diff
@@ -18,15 +18,26 @@ import * as child_process from 'child_process'
 import * as fs from 'fs-extra'
 
 const packageJsonFile = './package.json'
-// Create a backup so that we can restore the original later.
-fs.copyFileSync(packageJsonFile, `${packageJsonFile}.bk`)
+const webpackConfigJsFile = './webpack.config.js'
 
-const packageJson = JSON.parse(fs.readFileSync('./package.json', { encoding: 'UTF-8' }).toString())
-packageJson.version = '1.99.0-SNAPSHOT'
+try {
+    // Create backup files so we can restore the originals later.
+    fs.copyFileSync(packageJsonFile, `${packageJsonFile}.bk`)
+    fs.copyFileSync(webpackConfigJsFile, `${webpackConfigJsFile}.bk`)
 
-fs.writeFileSync(packageJsonFile, JSON.stringify(packageJson, undefined, '  '))
+    const packageJson = JSON.parse(fs.readFileSync(packageJsonFile, { encoding: 'UTF-8' }).toString())
+    packageJson.version = '1.99.0-SNAPSHOT'
```

Comment:
Does this strange version do anything? shouldn't we use the real version `-DEBUG-SNAPSHOT` or something more like that?

Reply:
it helps avoid vscode auto-update from the marketplace